extensionsGallery-insider.json

```diff
@@ -204,6 +204,57 @@
 				],
 				"statistics": [],
 				"flags": "preview"
+			},
+			{
+				"extensionId": "14",
+				"extensionName": "sql-search",
+				"displayName": "Redgate SQL Search",
+				"shortDescription": "Search across multiple databases",
+				"publisher": {
+					"displayName": "Redgate",
+					"publisherId": "Redgate",
+					"publisherName": "Redgate"
+				},
+				"versions": [
+					{
+						"version": "0.1.0",
+						"lastUpdated": "4/18/2018",
+						"assetUri": "",
+						"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+							{
+								"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+								"source": "https://foundrylabssqlsearch.blob.core.windows.net/sqlops-sql-search-insiders/sql-search-0.1.0.vsix"
```

Comment:
Let me sync with the team on this scenario.  Our current plan is that we should only directly link to VSIX content that is hosted under the Microsoft account for security and compliance reasons.  I think what we can probably do is copy this VSIX to our blob storage and give you a link that you can use here.

Reply:
No worries, bear in mind we could always add a download page for Insiders versions if the VSIX approach gets too complicated.